### PR TITLE
Introduce redis_store option (closes #1)

### LINKED ISSUE #1: configuration options
can you add something to the readme about how to configure this?  i want to change the session store to point to a redis box on another machine.  how do i do that?

### PR DESCRIPTION
**Pull request by kuraga**
/cc @radar

### CODE REVIEW

File: lib/rack/session/redis.rb [hunk 1]
```diff
@@ -14,7 +14,9 @@ def initialize(app, options = {})
         super
 
         @mutex = Mutex.new
-        @pool = ::Redis::Store::Factory.create @default_options[:redis_server]
+        @pool = @default_options.has_key?(:redis_store) ?
```
Comment (radar) [inline lib/rack/session/redis.rb]: This line is rather long. I think it'd be better split out over multiple lines and you could use `fetch` as well:

```ruby
@pool = @default_options.fetch(:redis_store,
  ::Redis::Store::Factory.create(@default_options[:redis_server]))
```

Comment (kuraga) [inline lib/rack/session/redis.rb]: But we can't evaluate second argument of this call unless `@default_options[:redis_server]` exists...

Comment (kuraga) [inline lib/rack/session/redis.rb]: @radar But we can't evaluate second argument of this call unless `@default_options[:redis_server]` exists...

Comment (radar) [inline lib/rack/session/redis.rb]: Ah, yes you're right.

**Comment by radar** [comment]
LGTM

**Pull request merged**